demo/config/system/system.txt

```diff
@@ -6,6 +6,7 @@ DECLARE_TARGET INST
 DECLARE_TARGET INST INST2
 DECLARE_TARGET EXAMPLE
 DECLARE_TARGET TEMPLATED
+DECLARE_TARGET DART
```

Comment:
This basically makes DART a part of the Demo now. Don't we need to modify the Gemfile to make sure we're including all the DART dependencies. I think you have it commented out right now.

Reply:
This adds a DART target, but it doesn't bring in all the gem requirements.   I'll add DONT_CONNECT to the interface so that it doesn't add noise if DART is not running or used.